src/libraries/Microsoft.Extensions.Logging/gen/DiagDescriptors.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Extensions.Logging.Generators
+{
+    internal static class DiagDescriptors
+    {
+        public static DiagnosticDescriptor InvalidLoggingMethodName { get; } = new (
+            id: "SYSLIB0013",
```

Comment:
Do we need to log these numbers as "used" in some central location?

[https://github.com/dotnet/runtime/blob/main/docs/project/list-of-obsoletions.md](https://github.com/dotnet/runtime/blob/main/docs/project/list-of-obsoletions.md)

Looks like some of these are already taken.

Reply:
Oh, I based off the increments from [this list](https://docs.microsoft.com/en-us/dotnet/core/compatibility/syslib-warnings/syslib0012).

- [ ] Seems like I need to start from `SYSLIB1000`

Reply:
The list you linked is what we shipped in 5.0. We have added more already in 6.0.

Reply:
@eerhardt in the link you shared it says:

"Currently the identifiers `SYSLIB0001` through `SYSLIB0999` are carved out for obsoletions. If we wish to introduce analyzer warnings not related to obsoletion in the future, we should begin at a different range, such as `SYSLIB2000`."

So I guess I'll start from SYSLIB1000